src/tinyformat.h

```diff
@@ -178,6 +179,18 @@ namespace tfm = tinyformat;
 
 namespace tinyformat {
 
+// Added for Bitcoin Core. Wrapper for checking format strings at compile time.
+// Unlike ConstevalFormatString this supports std::string for runtime string
+// formatting without compile time checks.
+template <unsigned num_params>
+struct FormatStringCheck {
```

Comment:
Checked and failures seem to be validated successfully from command line, but - unlike the previous versions - doesn't seem to be shown in the IDE... Weird :/